br/pkg/lightning/checkpoints/checkpoints_file_test.go

```diff
@@ -191,98 +185,110 @@ func (s *cpFileSuite) TestGet(c *C) {
 				}},
 			},
 		},
-	})
+	}
+	assert.Equal(t, expect, cp)
 
-	cp, err = s.cpdb.Get(ctx, "`db2`.`t3`")
-	c.Assert(err, IsNil)
-	c.Assert(cp, DeepEquals, &checkpoints.TableCheckpoint{
+	cp, err = cpdb.Get(ctx, "`db2`.`t3`")
+	require.NoError(t, err)
+	expect = &checkpoints.TableCheckpoint{
 		Status: checkpoints.CheckpointStatusLoaded,
 		Engines: map[int32]*checkpoints.EngineCheckpoint{
 			-1: {
 				Status: checkpoints.CheckpointStatusLoaded,
 				Chunks: []*checkpoints.ChunkCheckpoint{},
 			},
 		},
-	})
+	}
+	assert.Equal(t, expect, cp)
 
-	cp, err = s.cpdb.Get(ctx, "`db3`.`not-exists`")
-	c.Assert(cp, IsNil)
-	c.Assert(errors.IsNotFound(err), IsTrue)
+	cp, err = cpdb.Get(ctx, "`db3`.`not-exists`")
+	require.Nil(t, cp)
+	assert.True(t, errors.IsNotFound(err))
```

Comment:
```suggestion
	require.True(t, errors.IsNotFound(err))
```

We can consistently use `require` for fail fast. If the codebase exists two variants while no explicit benefits, it increases understanding overhead.

ditto others

Reply:
ok.